pyproject.toml

```diff
@@ -133,6 +133,7 @@ pytest-aiohttp = "^0.3.0"
 pytest-cov = "^3.0.0"
 pytest-env = "^0.6.2"
 types-PyYAML = "^6.0.1"
+flake8 = "^4.0.1"
```

Comment:
Please revert this change.

`rororo` aims to use `flake8` via `pre-commit`, so it shouldn't have be listed within dev dependencies.

To use `flake8` with `rororo`, you need to,

1. Install [pre-commit](https://pre-commit.com) (most appropriate way, imo, via `pipx install pre-commit`)
2. Enable `pre-commit` to the repository via `pre-commit install`
3. In order to use `flake8` in your editor find out path to `flake8` hook via [`pre-commit-which-hook-entry flake8`](https://pypi.org/project/pre-commit-run-hook-entry)